tests/console/check_system_info.pm

```diff
@@ -31,21 +31,21 @@ sub run {
     select_console('root-console');
     assert_script_run('setterm -blank 0') unless (check_var('ARCH', 's390x'));
 
-    assert_script_run("SUSEConnect --status-text > /dev/$serialdev", timeout => 180);
-
     if (!get_var('MILESTONE_VERSION')) {
         assert_script_run('cat /etc/issue');
     } else {
         my $milestone_version = get_var('MILESTONE_VERSION');
         assert_script_run("grep -w $milestone_version /etc/issue");
     }
-    assert_script_run('cat /etc/os-release');
+
+    script_run('zypper lr | tee /tmp/zypperlr.txt');
 
     my $myaddons = get_var('SCC_ADDONS', "");
-    $myaddons = $myaddons . ",base,serverapp"                             if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
-    $myaddons = $myaddons . ",base,desktop,we,python2"                    if (is_sle('15+') && check_var('SLE_PRODUCT', 'sled'));
-    $myaddons = $myaddons . ",base,serverapp,desktop,dev,lgm,python2,wsm" if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
-    $myaddons = $myaddons . ",python2"                                    if (is_sle('=15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
+    $myaddons = $myaddons . ",base,serverapp"                                  if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
+    $myaddons = $myaddons . ",base,desktop,we,python2"                         if (is_sle('15+') && check_var('SLE_PRODUCT', 'sled'));
+    $myaddons = $myaddons . ",base,serverapp,desktop,dev,lgm,python2,wsm"      if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
+    $myaddons = $myaddons . ",python2"                                         if (is_sle('=15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
+    $myaddons = $myaddons . ",base,serverapp,desktop,dev,lgm,python2,wsm,phub" if (get_var('HDDVERSION', "") =~ /leap/);
```

Comment:
you can use the ORIGIN_SYSTEM_VERSION or use the function is_leap_migration.
coz HDDVERSION will change after migration.